micall/utils/fetch_sequences.py

```diff
@@ -281,22 +281,27 @@ def check_hcv_seeds(project_config, unchecked_ref_names: set):
 downloaded from https://hcv.lanl.gov/content/sequence/NEWALIGN/align.html
 This script contains a complete list of the reference accession numbers.
 """)
-    source_ids = {'HCV-1a': 'Ref.1a.US.77.H77.NC_004102',
+    source_ids = {# 'HCV1A': 'NC_004102', SAME, all HCV1 seeds except B use this
+                  'HCV-1a': 'Ref.1a.US.77.H77.NC_004102',
+                   # 'HCV1B': 'AJ238799', DIFFERENT, used only for B
```

Comment:
How about adding a separate method to check HCV genotype refs?

Reply:
I've added a method to check our project genotype sequences against the online references - I think that's what you had in mind, right?

Reply:
That is what I had in mind, but now I see that the same sequences are used in `check_hcv_coordinates()`. If it's easy to combine those two methods, please do. If it's a pain for some reason, just print a message at the start of the new method to say that it's using the same sequences as `check_hcv_coordinates()`. One of the main uses for this script is for me to come back to, some time in the future, and remember where we got our reference sequences from and why we chose them.

Reply:
The new version looks much simpler. Merging.